render/container.go

```diff
@@ -23,7 +23,7 @@ const (
 // NB We only want processes in container _or_ processes with network connections
 // but we need to be careful to ensure we only include each edge once, by only
 // including the ProcessRenderer once.
-var ContainerRenderer = MakeFilter(
+var ContainerRenderer = ApplyDecorators(MakeFilter(
```

Comment:
Maybe it would be enough to only invoke ApplyDecorators in api_topologies, at the top level?